sage/pallet/src/mock.rs

```diff
@@ -204,7 +215,7 @@ pub struct MockAssetMediator;
 impl AssetManager for MockAssetMediator {
```

Comment:
I think it is important that the mock uses the sage-pallet here. We need to do this how we would do it in the actual runtime, so that we don't run into any unexpected type issues.